aptos/Move.toml

```diff
@@ -2,3 +2,14 @@
 authors = ["MoveFuns DAO contributors"]
 name = "movefuns"
 version = "1.0.0"
+
+[addresses]
+std = "0x1"
+aptos_std = "0x1"
+aptos_framework = "0x1"
+movefuns = "_"
+
+[dependencies]
+AptosFramework = { git = "https://github.com/aptos-labs/aptos-core.git", rev = "main", subdir = "aptos-move/framework/aptos-framework"}
```

Comment:
It would be better to use the mainnet branch?
Because some features of the main branch have not been moved to mainnet, there will be different versions of the  problem

If you choose a lower version, you may be able to reduce such situations

Reply:
Update to mainnet.